modules/cortico/Login/Login.js

```diff
@@ -0,0 +1,169 @@
+import { Ellipsis } from "../../Icons/Ellipsis";
+import "./Login.css";
+import { CorticoIcon } from "../../Icons/CorticoIcon";
+import { create, getCorticoUrl } from "../../Utils/Utils";
+
+export function addLoginForm(browser) {
+  const currentUser = localStorage.getItem("currentUser");
+
+  if (!getCorticoUrl()) {
+    alert("Please set the cortico url")
+
+    return
+  }
+
+  // if (getCorticoUrl() && currentUser == null)
+  document.body.appendChild(loginForm(browser));
+}
+
+export function loginForm(browser) {
+  const menuIcon = Ellipsis();
```

Comment:
@ferdinandpolpol the create function can now take HTML, so this can be much more compact. What do you think? 

See appointmentMenu() - https://github.com/cortico-health/oscar-enhancement-suite/commit/30f754302f88f6d7e6ce8ab20576dcb2853e4fd4#diff-9da2906a61ae6f666abebfae1a776a8bff911546ac43b04eca3967afa9fd55db

Reply:
Okay, I updated this branch to develop. Are we merging develop to main anytime soon?